src/visualize/image_like.jl

```diff
@@ -143,6 +143,16 @@ function _default{T<:Colorant, X}(img::Images.Image{T, 3, X}, s::Style, data::Di
 end
 
 """
+Displays 3D array as movie with 3rd dimension as time dimension
+"""
+function _default{T <: Colorant}(img::AbstractArray{T, 3}, s::Style, data::Dict)
+    # TODO axis array and stuff
+    video_signal = const_lift(play, img, 3, loop(1:size(img, 3)))
```

Comment:
I'll keep these in mind for a "clean" refactor for when things settle down a bit! There are quite a few things I'd like to change, whenever it's clearer that it's fine to completely drop support for the old Images.jl.